src/checkout/context/checkout.tsx

```diff
@@ -277,8 +288,6 @@ export const CheckoutProvider: FC<Props> = React.memo(({children}) => {
         }
 
         setCheckoutStatus(RemoteDataState.Done)
-        // Call the `quartz/me` endpoint to update the existing user metadata
-        dispatch(getQuartzMeThunk())
```

Comment:
having this here caused the page to crash since the user's status but they were still on the checkout page, which is forbidden for PAYG users